src/components/shell/deviceGroupDropdown/deviceGroupDropdown.js

```diff
@@ -9,7 +9,7 @@ import './deviceGroupDropdown.scss';
 
 export class DeviceGroupDropdown extends Component {
 
-  onChange = (deviceGroupIds) => ({ target: { value: { value } = {} } = {} }) => {
+  onChange = (deviceGroupIds) => (value) => {
```

Comment:
Is this only place where we get affected by the drop-down not getting selected correctly? Can we check if other places are fine?

Reply:
Yes I checked all other places. It's working fine.